src/pykeen/models/unimodal/hole.py

```diff
@@ -4,11 +4,9 @@
 
 from typing import Any, ClassVar, Mapping, Optional
 
-import torch
-
-from ..base import EntityRelationEmbeddingModel
+from ..nbase import ERModel
 from ...constants import DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE
-from ...moves import irfft, rfft
```

Comment:
can we delete this module completely yet?

Reply:
yes, cf. https://github.com/pykeen/pykeen/pull/953/commits/f812f3048a6275017f3212c9ddfbef260e1b0799